lib/tasks.js

```diff
@@ -588,25 +588,36 @@ module.exports = function(self, options) {
     console.log('... Done.');
   };
 
-  self.enableCleanPagesTreeTask = function() {
-    self.addTask('clean-pages-tree',
-      `Cleans your pages tree in the case the live one differs with the draft one,
-      The live tree will be modified based on the draft one.`,
-      self.cleanPagesTree
+  self.enableSyncPagesTreeTask = function() {
+    self.addTask('sync-page-tree',
+      `Synchronizes the published page tree with the draft one, across all locales.
+      If there is a discrepancy in the layout of the draft and published trees, the draft tree wins.`,
+      self.syncPagesTree
     );
   };
 
-  self.cleanPagesTree = async function() {
+  self.syncPagesTree = async function() {
     const locales = Object.keys(self.apos.workflow.locales)
       .filter((locale) => !locale.includes('-draft'));
 
     const pageTypes = self.apos.modules['apostrophe-pages'].options.types.map((type) => type.name);
```

Comment:
This will find only page types that are in the menu. This may not include all parked pages.

A better idea, to identify all pages, is to find all docs whose slug starts with `/`. This is a reliable test of page-ness in Apostrophe.

Reply:
Alright, I jsut modified it, just a question about trash pages:
* Should I sync trash pages too or just avoid them ?

Reply:
You should, since you don't want any sleeping problems with them to be reintroduced if they are removed from the trash.

Reply:
Sorry that wasn't clear, I was literally speaking about the page with a `/trash` slug 😄